common/azure-test-watcher/pom.xml

```diff
@@ -1,81 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-
-  <groupId>com.azure</groupId>
-  <artifactId>azure-test-watcher</artifactId>
```

Comment:
Reference needs to be removed from root ```pom.xml```, and also removed from versions text files.

Reply:
Done